bpython/repl.py

```diff
@@ -56,6 +58,7 @@
 
 
 class RuntimeTimer(object):
+    """Calculate running time"""
```

Comment:
Sure those sound much better! Ill make the changes you purposed and resubmit my pull request.